docs/README.md

```diff
@@ -0,0 +1,26 @@
+# Documentation
+
+Documentation for gnomad is generated using [Sphinx](https://www.sphinx-doc.org/en/master/).
```

Comment:
should we update gnomad to gnomad_qc (just to differentiate between this repo and gnomad_methods)?

Reply:
oops, yup, thank you!